tools/helm/livy/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM java:openjdk-8-jdk
-MAINTAINER Dalitso Banda <dalitsohb@gmail.com>
+FROM waterknight/spark:2.4.5
```

Comment:
can you add the required lines to this file instead of depending on your spark2.4.5?

Reply:
What lines do you refer? @mhamilton723

Reply:
I think that you don't want to depend on my Image. This image is built from spark Dockerfile.

I am going to remove that and push it again. Is enought with just pushing?

Reply:
@mhamilton723 Done!